README.md

```diff
@@ -44,7 +44,7 @@ The recommended way to use the KCL for Java is to consume it from Maven.
   <dependency>
       <groupId>software.amazon.kinesis</groupId>
       <artifactId>amazon-kinesis-client</artifactId>
-      <version>2.2.1</version>
+      <version>2.2.2</version>
```

Comment:
Minor - while we're here can we also update the 1.x version in the master README to be the most recent 1.11.2?

Reply:
addressing